great_expectations/data_context/store/ge_cloud_store_backend.py

```
@@ -21,6 +21,18 @@
 
 logger = logging.getLogger(__name__)
 
+SUPPORT_EMAIL = "support@greatexpectations.io"
```

I would shy away from including a hardcoded value like this in the library and instead make it a bit more vague like "please contact support." If we pull the error message from the payload and that has an email or link that can work.

---

https://github.com/great-expectations/great_expectations/pull/5797/files#r952817953

We could externalize this as an environment variable configuration field, but we need something like this in the event that the Cloud Service is down or totally unavailable.

---

Ah ok thanks for clarifying. Maybe instead of the email addr though point to the website where we will likely have a support button / section? Totally up to you and product for whatever the best approach is to keep friction low - just a suggestion.

---

I think that's a good idea.

@wookasz requested we provide a contact email, but I'm happy to update it to whatever we want, whether in this PR or in follow-up.

---

Okay with this for now but I'd like these values to come from the Cloud backend if at all possible (perhaps a refactor down the line).